__tests__/input-validator.test.ts

```diff
@@ -172,6 +173,33 @@ describe('force-avd-creation validator tests', () => {
   });
 });
 
+describe('emulator-port validator tests', () => {
+  it('Validates if emulator-port is even and in range', () => {
+    const func = () => {
+      validator.checkPort(5554);
+    };
+    expect(func).not.toThrow();
+  });
```

Comment:
Might be good to also test that the MAX_PORT value also is valid, since it's the other boundary value

Reply:
Isn't this already tested two tests later?

Reply:
Somewhat yes, but it uses the MAX_PORT variable directly, not its value, so changing MAX_PORT value would always make the test pass, however from the reqs we know that 5584 is the max and it will never change. Having such test and editing the MAX_PORT value would result in failure which is what we want